java/client/src/main/java/glide/api/models/commands/SortOptions.java

```diff
@@ -0,0 +1,53 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Builder;
+import lombok.RequiredArgsConstructor;
+
+@Builder
+public final class SortOptions {
+    public static final String LIMIT_COMMAND_STRING = "LIMIT";
+    public static final String ALPHA_COMMAND_STRING = "ALPHA";
+    public static final String STORE_COMMAND_STRING = "STORE";
+    private final Limit limit;
+    private final Order order;
+    private final boolean alpha;
+
+    @RequiredArgsConstructor
+    public static final class Limit {
+        private final Long offset;
+        private final Long count;
+    }
+
+    @RequiredArgsConstructor
+    public enum Order {
+        ASC("ASC"),
+        DESC("DESC");
+
+        private final String redisApi;
+    }
```

Comment:
You can use `toString()` and
```suggestion
    public enum Order {
        ASC,
        DESC,
    }
```